showCrime/showCrime/settings/development.py

```diff
@@ -0,0 +1,21 @@
+from .base import *
```

Comment:
Can we get rid of this file completely? I don't see a great need to print out settings every time the application starts.

Reply:
Will do.